test-alien-bind.js

```diff
@@ -0,0 +1,85 @@
+/* global describe, it, expect, beforeEach, afterEach */
+var sinon = require('sinon')
+
+describe('bind', function () {
+  var bind
+
+  beforeEach(function () {
+    sinon.spy(Function.prototype, 'bind')
+    bind = require('./alien-bind')
```

Comment:
True, though this still serves a purpose in that you setup the bind spy *before* the require so it does get tested. It would be better to do this for just one require for it to be more explicit but I can't think of a nice way, any ideas?